### ISSUE TITLE
MariaDB compatibility

### ISSUE THREAD

**Issue by Johanbos** [Johanbos/home-assistant-import#225]
The following changes need to be made when you use MariaDB on HA.
>    The first select query you use to get the list of items in HA needs to be modified.
    The database “statistics” needs to be written without the double quotes.
    The item start being named start in the first line needs to be named anything else then start. I used start1.

>   The queries created by the tool needs to be modified.
    The command unixepoch doesn’t exist in MariaDB
    Use UNIX_TIMESTAMP instead.

**Comment by Xavier82** [comment]
Is MariaDB mandatory to have installed to use the Home-Assistant-import withSQLite Web and the tool?

**Comment by Johanbos** [comment]
No, it is not mandatory. You have a choice to use the default or go with another vendor. 

https://community.home-assistant.io/t/what-s-the-best-database-for-ha-with-a-lot-of-entities/568631

**Comment by Xavier82** [comment]
Ok, great, thanks for your response.

**Comment by Johanbos** [comment]
Issue will not be solved, due to inactivity.
